sdk/eventhubs/azure-messaging-eventhubs-stress/src/main/java/com/azure/messaging/eventhubs/stress/util/TestUtils.java

```diff
@@ -43,10 +54,21 @@ public static BinaryData createMessagePayload(int messageSize) {
     }
 
     public static EventHubClientBuilder getBuilder(ScenarioOptions options) {
-        return new EventHubClientBuilder()
+        final EventHubClientBuilder builder = new EventHubClientBuilder()
             .connectionString(options.getEventHubsConnectionString())
             .retryOptions(new AmqpRetryOptions().setTryTimeout(Duration.ofSeconds(5)))
-            .eventHubName(options.getEventHubsEventHubName());
+            .eventHubName(options.getEventHubsEventHubName())
+            .transportType(options.getAmqpTransportType());
+
+        if (options.useV2Stack()) {
```

Comment:
nit, we should be able to specify it as system property in the chart.yaml, but I totally understand if you prefer writing Java code over helm templates 😅

Reply:
Ooooh. Let me do that. I had copied my workfrom the eh-track2-perf changes.